list_app.py

```diff
@@ -0,0 +1,12 @@
+cars = []
+num = int(input("How many cars do you want to add?\n"))
+
+while num != 0:
```

Comment:
This is a Great approach! What do you think about this tho?
```suggestion
while num > 0:
```

I feel like it would *help* us to prevent going into infinite loop in someway. If the project gets bigger and start making this file large or modifying variable `num`.👍